src/transport/smtp/client/tls.rs

```diff
@@ -12,20 +12,19 @@ use rustls::{
     ClientConfig, Error as TlsError, OwnedTrustAnchor, RootCertStore, ServerName,
 };
 
+/// TLS protocol versions.
+#[derive(Debug, Copy, Clone)]
+#[non_exhaustive]
+pub enum TlsVersion {
+    Tlsv10,
+    Tlsv11,
+    Tlsv12,
+    Tlsv13,
+}
```

Comment:
Shouldn't this be feature gated if no TLS features are turned on?

Reply:
Indeed, done.